packages/devtools_extensions/lib/src/template/_simulated_devtools_environment/connection_ui/_connect.dart

```diff
@@ -0,0 +1,220 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of '../_simulated_devtools_environment.dart';
+
+class _UriConnectionDisplay extends StatelessWidget {
```

Comment:
the git diff didn't do a good job here, but this file was moved from [_connect_ui.dart](https://github.com/flutter/devtools/pull/7133/files#diff-c0483a0033827e30ba1cc5d285d27073784ec9cd5bafa96c0f67b0b2ff06db56), and the classes were tweaked slightly to generalize them.